os/fs/mount/fs_mount.c

```diff
@@ -63,6 +63,7 @@
 #include <debug.h>
```

Comment:
Please add **is** to the statement, "a mount point created on it during boot time." --> "a mount point **is** created on it during boot time."